package/MDAnalysis/analysis/align.py

```diff
@@ -1350,45 +1350,51 @@ def get_atoms_byres(g, match_mask=np.logical_not(mismatch_mask)):
 
             # stop if we created empty selections (by removing ALL residues...)
             if ag1.n_atoms == 0 or ag2.n_atoms == 0:
-                errmsg = ("Failed to automatically find matching atoms: created empty selections. " +
-                          "Try to improve your selections for mobile and reference.")
+                errmsg = ("Failed to automatically find matching atoms: created empty selections. "
+                            "Try to improve your selections for mobile and reference.")
```

Comment:
You have now added too many spaces, please try to align your strings again...

Reply:
This is still to be fixed